src/sentry/static/sentry/app/components/events/interfaces/frame/frame.jsx

```diff
@@ -381,24 +384,36 @@ const RepeatedContent = styled(VertCenterWrapper)`
   justify-content: center;
 `;
 
-const NativeLineContent = styled(VertCenterWrapper)`
-  flex: 1;
-  overflow: hidden;
-  justify-content: center;
-
-  & > span {
-    display: block;
-    padding: 0 5px;
+const PackageInfo = styled('div')`
+  display: flex;
```

Comment:
Does this component have two display definitions?

Reply:
it's grid... flex was there by mistake :)